### PR TITLE
migration: added migration help field

### LINKED ISSUE #222: multivolumes dump
8564_y ->Url per volume
doi->0247_q, doi per volume
020__u ISBN per volume 
https://cds.cern.ch/record/2312909/export/hm?ln=en

create an dump intermediate file (as monograph) with a structure:
```
_migration: {
    children: [] 
}
```
where in the children we specify the list of recids connected with this multipart (usually it will be self)

when dumping documents, create a rule which will add fields to the dump of the document"
```
_migration:{
    has_multipart: True
    volumes: [{...}, ]
}
```
which indicates all the volumes belonging to the record currently and will be used to create multiple document when ingesting this dump to the database

### PR DESCRIPTION
**Pull request by kprzerwa**
( closes #222 )

### CODE REVIEW

File: cds_dojson/marc21/fields/books/multipart.py [hunk 1]
```diff
@@ -29,6 +29,58 @@
...
+    """Translates isbns stored in the record."""
+    _migration = self.get('_migration', {'volumes': []})
+    _isbns = self.get('isbns', [])
+
+    val_u = clean_val('u', value, str)
+    val_a = clean_val('a', value, str)
+
+    if val_u:
+        volume_search = \
+            re.search('(.*?)\(((v\.*|vol\.*|volume\.*|part)(\d+))\)', val_u)
```
Comment (equadon) [inline cds_dojson/marc21/fields/books/multipart.py]: Could there be extra spaces in the data, like having two spaces between the dot and the number `v.  1` or space between the `v` and the dot, e.g. `v . 1`? Or would it be possible that there is no dot at all or will all data be cleaned up before the migration is done?

Comment (kprzerwa) [reply cds_dojson/marc21/fields/books/multipart.py]: I didn't see those cases in the data so possible it does not happen, also this data is being currently cleaned by @agentilb .

File: cds_dojson/marc21/fields/books/multipart.py [hunk 2]
```diff
@@ -40,31 +92,38 @@ def title(self, key, value):
             }
 
 
-@model.over('volumes', '^246__')
-def volumes(self, key, value):
+@model.over('_migration', '^246__')
+def migration(self, key, value):
     """Translates volumes titles."""
```
Comment (ntarocco) [inline cds_dojson/marc21/fields/books/multipart.py]: I would maybe just add an extra comment to explain why this is needed: not records in CDS, records in ILS.

File: tests/test_series.py [hunk 3]
```diff
@@ -274,7 +295,119 @@ def test_monograph(app):
...
+                        'volume': 2
+                    },
+                    {
+                        'title': '1882-1905',
+                        'volume': 3
+                    },
+                ],
+                    'record_type': 'multipart',
+                },
+                "title": [{'title': 'Wissenschaftliche Abhandlungen'}],
```
Comment (ntarocco) [inline tests/test_series.py]: shouldn't it be:
```
title: {
    title: ''
}
```
?
is it a list?

Comment (kprzerwa) [reply tests/test_series.py]: ah, it is a leftover from serials (they can have list of titles in temporary files as book might be a part of many serials) I will fix it for multiparts

**Pull request merged**